pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1281,13 +1281,46 @@ func (self *AWSCloud) TCPLoadBalancerExists(name, region string) (bool, error) {
 	return false, nil
 }
 
+// Retrieves instance's vpc id from metadata
+func (self *AWSCloud) findVPCId() (string, error) {
+
+	metadata := self.awsServices.Metadata()
+	macs_bytes, err := metadata.GetMetaData("network/interfaces/macs/")
```

Comment:
I believe the official k8s/go style is macsBytes: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps